dev-libs/opencryptoki/opencryptoki-3.23.0.ebuild

```
@@ -0,0 +1,103 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic
+
+DESCRIPTION="PKCS#11 provider cryptographic hardware"
+HOMEPAGE="https://github.com/opencryptoki/opencryptoki"
+SRC_URI="https://github.com/opencryptoki/opencryptoki/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P}"
+
+# Upstream is looking into relicensing it into CPL-1.0 entirely; the CCA
+# token sources are under CPL-1.0 already.
+LICENSE="CPL-0.5"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~s390 ~x86"
+IUSE="debug +tpm"
+
+DEPEND="
+	net-nds/openldap
+	tpm? ( app-crypt/trousers )
+	>=dev-libs/openssl-1.1.0:0=
```

We like to sort unconditionals before conditional deps.
```suggestion
	>=dev-libs/openssl-1.1.0:0=
	net-nds/openldap
	tpm? ( app-crypt/trousers )
```